### ISSUE TITLE
Accessibility Linting

### ISSUE THREAD

**Issue by meiamsome** [CodingTrain/project-trainsite-frontend#3]
I think it would be great to get some linting for common accessibility problems, but I am not aware of any good React tools. If anyone knows of some, it would be good to post them here, preferably with pros & cons

**Comment by Fraasi** [comment]
Oh my, you've been busy during the night 😄   
Yes, it would be good to catch these while developing.  
Maybe [react-a11y](https://github.com/reactjs/react-a11y) is our choice?
Edit: meant [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)  
Or maybe both as it says in the readme: This project does not replace react-a11y, but can and should be used in conjunction with it.

**Comment by phucduong86** [comment]
[React-axe](https://github.com/dequelabs/react-axe) looks like a great candidate too!
It is recommended on react documentation : https://reactjs.org/docs/accessibility.html

**Comment by Fraasi** [comment]
So I made a PR for this. It was very simple to setup(also very simple to remove if we so decide later). React-axe has really nice error reporting and easy to turn of when not needed. After testing I liked it and made the PR. Your thoughts?

**Comment by phucduong86** [comment]
Looks great @Fraasi
